fee_calculator/apps/calculator/management/commands/lgfs_2022_uplift.py

```diff
@@ -0,0 +1,107 @@
+from django.core.management import BaseCommand
+
+from calculator.models import (Scheme, Price)
+
+import csv
+
+
+def csv_to_json(csvFilePath):
+    jsonArray = []
+
+    with open(csvFilePath, encoding='utf-8-sig') as csvf:
+        csvReader = csv.DictReader(csvf)
+        for row in csvReader:
+            jsonArray.append(row)
+    return jsonArray
```

Comment:
could make jsonArray, csvFilePath and csvReader snake_case like the rest of the code for consistency

Reply:
Guess that gave away the fact that I just copied and pasted that bit of code from Stack Overflow 😅